Automattic/jetpack
# GIF Block: Accept Giphy Shortlinks

Fixes #11848

## Proposed changes:

* Accepts Giphy shortlinks for the GIF block as a valid URL. 

Unfortunately, I've checked the [API documentation](https://developers.giphy.com/branch/master/docs/api/endpoint/#get-gif-by-id) and it isn't possible to directly access a GIF's data from the shortlink. This means needing to do an extra request to acquire this information, but that shouldn't be too big a deal. 

## Jetpack product discussion


See original issue.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Insert the GIF block into the Editor
* Use the following shortlink, or any of your choosing: https://gph.is/g/aKnlLW3
* Confirm that it embeds correctly, and that you can change this URL to another shortlink: https://gph.is/2wQDf7Y

<img width="905" alt="Screenshot 2023-12-27 at 11 28 30" src="https://github.com/Automattic/jetpack/assets/43215253/93fc7387-18d5-41c8-901b-97acd13f5b04">
